package proto_ff;

import "yd_fieldoptions.proto";


message tasktrackertasktracker
{
	optional int32 comID = 1[(yd_fieldoptions.field_cname) = "完成条件类型ID"];
	optional int32 trueID = 2[(yd_fieldoptions.field_cname) = "实际完成条件类型ID"];
}


message Sheet_tasktrackertasktracker
{
	repeated tasktrackertasktracker tasktrackertasktracker_List = 1[(yd_fieldoptions.field_arysize)=139];
}
